docs/proposals/client-settings.md

```diff
@@ -100,6 +88,12 @@ type ClientSettingsPolicyConfig struct {
     KeepAlive *ClientKeepAlive `json:"keepAlive,omitempty"`
 }
 
+type ClientSettingsPolicyStatus struct {
+    // Conditions describe the current conditions of the ClientSettingsPolicy.
+    // +optional
+    Conditions []metav1.Condition `json:"conditions,omitempty"`
```

Comment:
Should we update this to match https://github.com/nginxinc/nginx-gateway-fabric/pull/1793/files#diff-f30353a8c42c8b931fd93c8bdceabb5e0ac494a351b71a2b903aebdd239c2c95R64?

Reply:
yep, somehow missed this. Thanks for catching!